core/src/test/kotlin/in/specmatic/core/pattern/DateTimePatternTest.kt

```diff
@@ -48,14 +48,20 @@ internal class DateTimePatternTest {
 
     @Test
     fun `should match RFC3339 date time format`() {
```

Comment:
Should convert this to a parameterised test? and also have one or two tests for non RFC 3339 compliant dates so that we can verify the parse fails?

Reply:
Pushed the relevant changes. Build is in progress.